config/jobs/kubernetes/minikube/minikube.yaml

```diff
@@ -0,0 +1,32 @@
+presets:
+- labels:
+    preset-k8s-ssh: "true"
+  env:
+  - name: PULL_ID
+    value: $(PULL_NUMBER)
```

Comment:
hmm not sure if this will work.

Reply:
@krzyzacy @cjwagner who've more recently touched the presets

Reply:
we already have `preset-k8s-ssh: "true"` in https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L491 which means this need to be named something else :-)

also `PULL_NUMBER` is already an env you can consume in your container so you don't have to make another env?